app/routes/crate/version.js

```diff
@@ -13,7 +13,7 @@ export default Route.extend({
   async model(params) {
     const requestedVersion = params.version_num;
     const crate = this.modelFor('crate');
-    const maxVersion = crate.get('max_version');
+    const maxVersion = crate.max_version;
```

Comment:
I'd prefer to keep `.get` for (ED) models since they're proxy objects.

Reply:
that's not quite correct AFAIK. async relationships are proxy objects, but models and their regular attributes are just regular things and don't need to use `.get()` anymore.